backend/api/models.py

```diff
@@ -171,3 +171,13 @@ class Stats(models.Model):
     confidence_low = models.FloatField(blank=True, null=True)
     confidence_high = models.FloatField(blank=True, null=True)
     confidence_level = models.FloatField(blank=True, null=True)
+
+
+class Uptake(models.Model):
+    type = models.CharField(max_length=100)
+    experiment = models.CharField(max_length=255)
+    branch = models.CharField(max_length=255)
```

Comment:
Branch needs to also allow null and blank becuase it could be missing.

Reply:
Good catch. I'll update.